packages/universal-router-sdk/package.json

```diff
@@ -48,10 +48,9 @@
     "@pancakeswap/permit2-sdk": "workspace:*",
     "@pancakeswap/sdk": "workspace:*",
     "@pancakeswap/smart-router": "workspace:*",
-    "@pancakeswap/swap-sdk-core": "workspace:*",
     "@pancakeswap/v3-sdk": "workspace:*",
```

Comment:
`import { Percent } from '@pancakeswap/swap-sdk-core'`

'@pancakeswap/swap-sdk-core' still in used